nilearn/plotting/html_connectome.py

```diff
@@ -319,25 +319,25 @@ def view_connectome(adjacency_matrix, node_coords, edge_threshold=None,
     return _make_connectome_html(connectome_info)
 
 
-def view_markers(marker_coords, marker_color='auto', marker_size=5.,
-                 marker_labels=None, title=None, title_fontsize=25):
-    """Insert a 3d plot of markers in a brain into an HTML page.
+def view_nodes(node_coords, node_color='auto', node_size=5.,
```

Comment:
We need to deprecate `view_markers` (see my comment in the issue).

Reply:
Yes, I forgot about it sorry. Will take care of this now!